apps/api/src/services/tag/tag.service.ts

```diff
@@ -17,18 +18,33 @@ export class TagService {
     });
   }
 
-  public async getInUseByUser(userId: string) {
-    return this.prismaService.tag.findMany({
+  public async getTagsForUser(userId: string) {
+    const tags = await this.prismaService.tag.findMany({
       orderBy: {
         name: 'asc'
       },
+      include: {
+        _count: {
+          select: {
+            orders: true
+          }
+        }
+      },
       where: {
-        orders: {
-          some: {
+        OR: [
+          {
             userId
+          },
+          {
+            userId: null
           }
-        }
+        ]
       }
     });
+
+    return tags.map((tag) => ({
```

Comment:
With this, you don't need `omit()`:

```
return tags.map(({ _count, id, name }) => ({
  id,
  name,
  isUsed: _count.orders > 0
}));
```

Reply:
7f80738

Reply:
removing unnecessary `omit` import - e0f7357
missing userId in return - 877fd87

Reply:
I initially chose omit because it's not explicit, meaning that the tag model can have other properties that don't need to be specified in the return. 
For example the user interface defines tags like this:
```ts
tags: (Tag & { isUsed: boolean })[];
```
But the tag model probably won't be changed as often so your suggestion makes more sense I guess.